src/main/surveys/surveyDialog.js

```diff
@@ -0,0 +1,245 @@
+/**
+ * The survey dialog component
+ *
+ * Contains a component for initializing the survey `BrowserWindow` based on the
+ * instructions of the `dialogManager`.
+ * Copyright 2017 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ * The research leading to these results has received funding from the European Union's
+ * Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+"use strict";
+
+var fluid = require("infusion"),
+    electron = require("electron"),
+    ipcMain = electron.ipcMain,
+    gpii = fluid.registerNamespace("gpii");
+
+require("../dialog.js");
+
+// XXX: Needed to circumvent the certificate error for the "umd.edu" domain.Should
```

Comment:
Create JIRA for this (and reference)

Reply:
The ticket for this issue is https://issues.gpii.net/browse/GPII-2818.